content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -0,0 +1,325 @@
+
+{{< command >}}
+$ npm init -y
+$ npm install -D serverless serverless-localstack serverless-deployment-bucket
+{{< / command >}}
+
+The `serverless-localstack` plugin allows your Serverless project to redirect AWS API calls to LocalStack. The `serverless-deployment-bucket` plugin creates a deployment bucket in LocalStack, which stores the deployment artifacts to ensure that old deployment buckets don't linger around after a deployment. We can now set up the plugin by adding the following properties to our `serverless.yml` file:
+
+```yaml
+...
+plugins:
+  - serverless-deployment-bucket
+  - serverless-localstack
+
+custom:
+  localstack:
+    stages:
+      - local
+```
+
+This sets up Serverless to use the LocalStack plugin but only for the stage **local**. To ensure that our Serverless project deploys only on LocalStack, not real AWS Cloud, we need to set a `--stage` flag with the `serverless deploy` command and set the flag variable to `local`. 
+
+You can also configure `serverless deploy --stage local` as a `deploy` script in your `package.json` to allow you to run the `serverless deploy` command directly over your local infrastructure. The `package.json` file should look like this:
+
+```json
+{
+  "name": "serverless-elb",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "scripts": {
+    "deploy": "sls deploy --stage local"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "serverless": "^3.25.0",
+    "serverless-deployment-bucket": "^1.6.0",
+    "serverless-localstack": "^1.0.1"
+  }
+}
+```
+
+You can give the above setup a run by deploying your Serverless project and checking the Lambda function that has been created via our existing Serverless project:
+
+{{< command >}}
+$ npm run deploy
+$ awslocal lambda list-functions
+
+{
+    "Functions": [
+        {
+            "FunctionName": "serverless-elb-local-hello",
+            "FunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:serverless-elb-local-hello",
+            "Runtime": "nodejs12.x",
+            "Role": "arn:aws:iam::000000000000:role/serverless-elb-local-us-east-1-lambdaRole",
+            "Handler": "handler.hello",
+            ...
+        }
+    ]
+}
+{{< / command >}}
+
+We can now create our target Lambda functions to work with Node.js runtime and configure ELB Application Load Balancers via our `serverless.yml` configuration.
+
+## Create Lambda functions & ELB Application Load Balancers
+
+Let us now create two Lambda functions, named `hello1` and `hello2`, which can run over Node.js 12.x runtime. Open `handler.js` and replace the existing code with the following:
+
+```js
+'use strict';
+
+module.exports.hello1 = async (event) => {
+  console.log(event);
+  return {
+    "isBase64Encoded": false,
+    "statusCode": 200,
+    "statusDescription": "200 OK",
+    "headers": {
+        "Content-Type": "application/json"
+    },
+    "body": "Hello 1"
```

Comment:
Ideally we should return a JSON content here in the `body`, if we use `application/json` content type. Alternatively, we could also remove the `Content-Type` header, or change it to `text/plain`.

Same in line 152 below.